db/seeds/dev/cardgames.js

```diff
@@ -0,0 +1,25 @@
+const cardGamesData = require('../../data/cardGamesData.js');
+
+const createCardGame = async (knex, game) => {
+  const newCard = await knex('card_games').insert({
+    name: game.name,
+    instructions: game.instructions,
+    description: game.description,
+    materials: game.materials,
+    number_of_players: game.number_of_players,
+    video: game.video,
+    family_friendly: game.family_friendly
+  });
+}
+
+exports.seed = async (knex) => {
+  try {
+    await knex('card_games').del();
+    let gamePromises = cardGamesData.map(game => {
+      return createCardGame(knex, game);
+    })
+    return Promise.all(gamePromises)
+  } catch(error) {
+    console.log(`Error seeding data: ${error}`)
+  }  
+};
```

Comment:
Looking at this part, correct me if I am wrong but I don't think we need the `async` `await` syntax since we are not dealing with promises. Is this correct?

Reply:
You're right! Thanks for catching that.  I'll update this and push up the changes before it's merged.